server/test/repository/ApiBridgeConfigurationRepositoryTest.java

```diff
@@ -0,0 +1,151 @@
+package repository;
+
+import static models.ApiBridgeConfigurationModel.ApiBridgeDefinition;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static services.apibridge.ApiBridgeServiceDto.CompatibilityLevel;
+
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import models.ApiBridgeConfigurationModel;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ApiBridgeConfigurationRepositoryTest extends ResetPostgres {
+
+  private ApiBridgeConfigurationRepository repo;
+
+  @Before
+  public void setUp() {
+    repo = instanceOf(ApiBridgeConfigurationRepository.class);
+  }
+
+  @Test
+  public void findAll_succeeds() {
+    repo.insert(createBridgeConfigurationModel().setAdminName("admin-name-1"))
+        .toCompletableFuture()
+        .join();
+    repo.insert(
+            createBridgeConfigurationModel().setAdminName("admin-name-2").setUrlPath("urlPath2"))
+        .toCompletableFuture()
+        .join();
+    repo.insert(
+            createBridgeConfigurationModel().setAdminName("admin-name-3").setUrlPath("urlPath3"))
+        .toCompletableFuture()
+        .join();
+
+    var model = repo.findAll().toCompletableFuture().join();
+    assertThat(model).hasSize(3);
+  }
+
+  @Test
+  public void findByHostUrlAndUrlPathAndCompatibilityLevel_succeeds() {
+    ApiBridgeConfigurationModel model =
+        repo.insert(createBridgeConfigurationModel()).toCompletableFuture().join();
+    ApiBridgeConfigurationModel persistedModel = refetchModelFromDb(model);
+    assertThat(persistedModel).usingRecursiveComparison().isEqualTo(model);
+  }
+
+  @Test
+  public void insert_persists() {
```

Comment:
question: How is this different than the test above?

Reply:
I don't think we need both, but you could add a comment that's it's also testing the insert flow.